protos/sources/vega/governance.proto

```diff
@@ -154,6 +154,8 @@ message NewMarketConfiguration {
   string tick_size = 16;
   // If enabled aggressive orders sent to the market will be delayed by the configured number of blocks
   bool enable_transaction_reordering = 17;
+  // Number of allowed price levels between an AMM's fair price and its quote prices. An AMM definition that exceeds this will we rejected at submission.
```

Comment:
```suggestion
  // Number of allowed price levels between an AMM's fair price and its quote prices. An AMM definition that exceeds this will be rejected at submission.
```